integreat_cms/cms/views/region_condition/region_condition_view.py

```diff
@@ -20,3 +31,38 @@ class RegionConditionView(TemplateView):
     template_name = "region_condition/region_condition.html"
 
     extra_context = {"current_menu_item": "region_condition"}
+
+    def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
+        r"""
+        Render region list
+
+        :param request: The current request
+        :param \*args: The supplied arguments
+        :param \**kwargs: The supplied keyword arguments
+        :return: The rendered template response
+        """
+
+        regions = Region.objects.all()
+        query = None
+
+        search_data = kwargs.get("search_data")
```

Comment:
Where do I enter this? 😭

Reply:
Not yet at all. I was think that it might make sense in the future, but maybe it makes more sense to remove this again until it's actually implemented :sweat_smile:

Reply:
Ah, OK! No, leave it in!